### PR TITLE
[FLINK-19430][docs-zh] Translate page datastream_tutorial into Chinese

### PR DESCRIPTION
**Pull request by wangxlong**
## What is the purpose of the change

Translate page datastream_tutorial into Chinese

## Brief change log

Translate page datastream_tutorial into Chinese, The doc is located at /dev/python/user-guide/datastream_tutorial.zh.md

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no )
  - The runtime per-record code paths (performance sensitive): (no)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: ( no)

## Documentation

  - Does this pull request introduce a new feature? (no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by wangxlong** [comment]
Hi @dianfu, Could you help have a review in your free time, Thank you~

File: docs/dev/python/datastream_tutorial.zh.md [hunk 1]
```diff
@@ -22,76 +22,77 @@ specific language governing permissions and limitations
...
+你现在可以在这个数据流上执行转换操作，或者使用 _sink_ 将数据写入外部系统。本教程使用 `StreamingFileSink` 将数据写入 `/tmp/output` 文件目录中。
 
 {% highlight python %}
 ds.add_sink(StreamingFileSink
     .for_row_format('/tmp/output', SimpleStringEncoder())
     .build())
 {% endhighlight %}
 
-The last step is to execute the actual PyFlink DataStream API job. PyFlink applications are built lazily and shipped to the cluster for execution only once fully formed. To execute an application, you simply call `env.execute(job_name)`.
+最后一步是执行真实的 PyFlink DataStream API 作业。PyFlink applications 是惰性构建的，并且只有完全构建之后才会提交给集群执行。要执行一个应用程序，你只需简单的调用 `env.execute(job_name)`。
```
Comment (dianfu) [inline docs/dev/python/datastream_tutorial.zh.md:89]:
```suggestion
最后一步是执行真实的 PyFlink DataStream API 作业。PyFlink applications 是懒加载的，并且只有在完全构建之后才会提交给集群上执行。要执行一个应用程序，你只需简单地调用 `env.execute(job_name)`。
```

File: docs/dev/python/datastream_tutorial.zh.md [hunk 2]
```diff
@@ -116,28 +117,28 @@ if __name__ == '__main__':
...
+最后，你可以在命令行上看到执行结果:
 
 {% highlight bash %}
 $ find /tmp/output -type f -exec cat {} \;
 1,aaa
 2,bbb
 {% endhighlight %}
 
-This walkthrough gives you the foundations to get started writing your own PyFlink DataStream API programs. To learn more about the Python DataStream API, please refer to [Flink Python API Docs]({{ site.pythondocs_baseurl }}/api/python) for more details.
+本教程为你开始编写自己的 PyFlink DataStream API 程序提供了基础。为了了解更多关于 Python DataStream API 的使用，请查阅 [Flink Python API Docs]({{ site.pythondocs_baseurl }}/api/python)。
```
Comment (dianfu) [inline docs/dev/python/datastream_tutorial.zh.md:144]:
```suggestion
本教程为你开始编写自己的 PyFlink DataStream API 程序提供了基础。如果需要了解更多关于 Python DataStream API 的使用，请查阅 [Flink Python API Docs]({{ site.pythondocs_baseurl }}/api/python)。
```

**Review by dianfu — COMMENTED** [review]
@wangxlong Thanks for the PR! Have left a few comments. As most of the content of this page is similar to https://ci.apache.org/projects/flink/flink-docs-master/zh/try-flink/datastream_api.html, you can refer to it when translating this page.

**Comment by wangxlong** [comment]
@dianfu  Thank you for reminder and patient review. Updated~

**Review by dianfu — APPROVED** [review]
LGTM

**Pull request merged**